### ISSUE TITLE
NTR: alpha-glucosaminidase high in blood (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8236]
New term request alpha-glucosaminidase high in blood (elthat_190712173900):   UMLS: 5195140

**Comment by pnrobinson** [comment]
@MickeySegal

Is this N-acetyl-alpha-D-glucosaminidase - the enzyme affected in Mucopolysaccharidosis type IIIB (Sanfilippo B)? Inwhat situation is this enzyme increased?

**Comment by MickeySegal** [comment]
It is "alpha-N-acetylglucosaminidase" elevated in "Spondyloepiphyseal dysplasia, Kondo-Fu type":
https://www.ncbi.nlm.nih.gov/books/NBK597768/
"Increased activity of lysosomal enzymes including alpha-N-acetylgalactosaminidase, alpha-N-acetylglucosaminidase, beta-glucuronidase, total beta-hexosaminidases, hexosaminidase A (MUGS substrate), and iduronate-2-sulfatase in plasma, but normal activity in leukocytes"

And I see that we hadn't listed to acetylgalactosaminidase finding - it would be good to get a code for that too.

**Comment by pnrobinson** [comment]
@MickeySegal 
is this what was missing?
Reduced tissue alpha-N-acetylglucosaminidase activity HP:6000360

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000971
- label: Elevated tissue alpha-N-acetylglucosaminidase activity
- parents: [Abnormal tissue enzyme concentration or activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:30046013
- def: Concentration or activity of alpha-N-acetylglucosaminidase (EC 3.2.1.50) above the upper limit of normal. This enzyme can be measured in multiple tissues including leukocytes and cultured fibroblasts. It is also known as alpha-acetylglucosaminidase, N-acetyl-alpha-D-glucosaminidase, N-acetyl-alpha-glucosaminidase, and alpha-D-2-acetamido-2-deoxyglucosidase.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8236
- synonyms:

**Comment by pnrobinson** [comment]
See this:
Reduced alpha-N-acetylgalactosaminidase activity in cultured fibroblasts HP:4000202
perhaps it should bhe converted to "tissue", check this.

**Comment by pnrobinson** [comment]
done

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK597768/ said that the alpha-N-acetylglucosaminidase elevation was 
"in plasma, but normal activity in leukocytes".  So tissue seems like the wrong choice.

And https://hpo.jax.org/browse/term/HP:4000202 is for reduced levels and we want elevation, also in plasma.
